src/main/java/com/sobytylnik/hello/ExceptionListPackage/EntityAlreadyExistsException.java

```diff
@@ -0,0 +1,7 @@
+package com.sobytylnik.hello.ExceptionListPackage;
```

Comment:
Illegal package name. Rename to `com.sobytylnik.exception`.
It's better to work not in `com.sobytylnik.hello`, but in `com.sobytylnik`